### PR TITLE
Feature/integration

### PR DESCRIPTION
**Pull request by tkieslich**
Gallery added

### CODE REVIEW

File: Configuration/TypoScript/Structure/Gridelements.typoscript [hunk 1]
```diff
@@ -0,0 +1,21 @@
+lib.gridelements.defaultGridSetup {
```
Comment (albig) [inline Configuration/TypoScript/Structure/Gridelements.typoscript]: Do we really need gridelements? If so, please add it to the dependancies in the composer.json.

File: ext_localconf.php [hunk 2]
```diff
@@ -17,6 +17,10 @@ function () {
             '<INCLUDE_TYPOSCRIPT: source="DIR:EXT:' . $extensionKey . '/Configuration/TsConfig/Page/" extensions="tsconfig">'
         );
 
+        \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addPageTsConfig(
```
Comment (albig) [inline ext_localconf.php]: We usually prefer to include this via registerPageTSConfigFile() in TCA/Overrides/pages.php. But never mind. We will change this to our needs after full feature integration.

File: Configuration/TypoScript/Structure/Gridelements.typoscript (hunk 1 above)
Comment (tkieslich) [reply Configuration/TypoScript/Structure/Gridelements.typoscript]: unfortunately yes, better working solutions will come in 10. I will move the composer Entry.

File: ext_localconf.php (hunk 2 above)
Comment (tkieslich) [reply ext_localconf.php]: No Problem, i can change.

**Pull request merged**